Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -80,7 +80,7 @@
             android:name=".contactdiary.ui.ContactDiaryActivity"
             android:exported="false"
             android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.NoActionBar" />
+            android:theme="@style/Theme.MaterialComponents.NoActionBar" />
```

Comment:
This makes the UI related to the diary activity dark on my phones, even if I have a light theme. My Huawei doesn't even have a dark theme but I get dark screens anyway.